pkg/agent/controller/egress/egress_controller.go

```diff
@@ -1118,39 +1127,42 @@ func (c *EgressController) syncEgress(egressName string) error {
 	}()
 
 	egressIP := net.ParseIP(eState.egressIP)
-	// Install SNAT flows for desired Pods.
-	for pod := range pods {
-		eState.pods.Insert(pod)
-		stalePods.Delete(pod)
+	failurePolicy := egress.Spec.FailurePolicy
+	if failurePolicy == crdv1b1.FailurePolicyDrop || egressNodeAvailability(egress) {
```

Comment:
This doesn't seem dropping the traffic but still sending it to the egress IP. This is more like `Ignore` instead of `Drop`.

Reply:
yes we are kind of Ignoring the packet